model/src/main/java/uk/gov/pay/commons/model/SupportedLanguageJsonDeserialize.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.pay.commons.model;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+
+public final class SupportedLanguageJsonDeserialize extends JsonDeserializer<SupportedLanguage> {
```

Comment:
can it be called `SupportedLanguageJsonDeserializer`

Reply:
Good point, has been renamed.